src/styles/components/breadcrumbs.less

```diff
@@ -142,3 +142,11 @@ Breadcrumbs
     }
   }
 }
+
+.breadcrumb-style-headline {
+  margin-bottom: @base-spacing-unit * 2/3;
+  .h4 {
+    font-size: @base-spacing-unit * .65;
```

Comment:
yeah same with `line-height` below this too. i believe `base-spacing-unit` is used for margin/padding only